src/components/Notifications/form/event.ts

```diff
@@ -1,5 +1,4 @@
 import { SpruceFormProps } from "components/SpruceForm/types";
-import { AntdSelect } from "components/SpruceForm/Widgets/AntdWidgets";
```

Comment:
There's a selectable blank area in the `Select`s which should probably not appear (this also affects other `Select`s in other forms)

<img width="438" alt="Screen Shot 2022-12-12 at 12 29 46 PM" src="https://user-images.githubusercontent.com/47064971/207113292-8e062fca-7df3-47c8-bbc1-5a0b6951bb4a.png">

<img width="426" alt="Screen Shot 2022-12-12 at 12 42 07 PM" src="https://user-images.githubusercontent.com/47064971/207115846-e7718d09-b706-400c-9963-88d90cef0384.png">